packages/cli-repl/test/e2e.spec.ts

```diff
@@ -29,8 +29,8 @@ describe('e2e', function() {
 
     beforeEach(async() => {
       dbName = `test-${Date.now()}`;
-      const connectionString = `mongodb://localhost:27018/${dbName}`;
-      shell = TestShell.start({ args: [ connectionString ] });
+      const dbConnectionString = `${connectionString}}/${dbName}`;
```

Comment:
I think this can just be the connectionString from `startTestServer` since concatenating is probably not going to work if we use atlas or some url that has query parameters

Reply:
Ok, will get rid of the db in the uri